modules/serverless-kafka-source-yaml.adoc

```diff
@@ -0,0 +1,59 @@
+// Module included in the following assemblies:
+//
+// * serverless/event_sources/serverless-kafka-source.adoc
+
+[id="serverless-kafka-source-yaml_{context}"]
+= Creating a Kafka event source using YAML
+
+You can create a Kafka event source by using YAML.
```

Comment:
I'd prefer not to, since we'd need to try to make all the other bits in the book that mention YAML the same then, and afaik they're all just "by using YAML", which I think is OK?

Reply:
Sounds good to me.